diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b..f9d297d0 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+COMPANY_FULLNAME=The Ember Authors
+COMPANY_SHORTNAME=The Ember Authors
+PRODUCT_FULLNAME=Ember
+PRODUCT_SHORTNAME=Ember
+PRODUCT_INSTALLER_FULLNAME=Ember Installer
+PRODUCT_INSTALLER_SHORTNAME=Ember Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Ember Authors. All rights reserved.
+MAC_BUNDLE_ID=dev.v11exe.ember
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
diff --git a/chrome/install_static/chromium_install_modes.h b/chrome/install_static/chromium_install_modes.h
index f29a8228..e2e7a52f 100644
--- a/chrome/install_static/chromium_install_modes.h
+++ b/chrome/install_static/chromium_install_modes.h
@@ -17,14 +17,14 @@ namespace install_static {
 
 // The brand-specific company name to be included as a component of the install
 // and user data directory paths. May be empty if no such dir is to be used.
-inline constexpr wchar_t kCompanyPathName[] = L"";
+inline constexpr wchar_t kCompanyPathName[] = L"v11exe";
 
 // The brand-specific product name to be included as a component of the install
 // and user data directory paths.
-inline constexpr wchar_t kProductPathName[] = L"Chromium";
+inline constexpr wchar_t kProductPathName[] = L"Ember";
 
 // The brand-specific safe browsing client name.
-inline constexpr char kSafeBrowsingName[] = "chromium";
+inline constexpr char kSafeBrowsingName[] = "ember";
 
 // Note: This list of indices must be kept in sync with the brand-specific
 // resource strings in chrome/installer/util/prebuild/create_string_rc.
@@ -43,36 +43,35 @@ inline constexpr auto kInstallModes = std::to_array<InstallConstants>({
         .install_suffix =
             L"",  // Empty install_suffix for the primary install mode.
         .logo_suffix = L"",  // No logo suffix for the primary install mode.
-        .app_guid =
-            L"",  // Empty app_guid since no integration with Google Update.
-        .base_app_name = L"Chromium",              // A distinct base_app_name.
-        .base_app_id = L"Chromium",                // A distinct base_app_id.
-        .browser_prog_id_prefix = L"ChromiumHTM",  // Browser ProgID prefix.
+        .app_guid = L"{FDFEA5B9-ECF4-4569-9E22-16FDB96870AB}",
+        .base_app_name = L"Ember",              // A distinct base_app_name.
+        .base_app_id = L"Ember",                // A distinct base_app_id.
+        .browser_prog_id_prefix = L"EmberHTM",  // Browser ProgID prefix.
         .browser_prog_id_description =
-            L"Chromium HTML Document",  // Browser ProgID description.
-        .direct_launch_url_scheme = "chromium",
-        .pdf_prog_id_prefix = L"ChromiumPDF",  // PDF ProgID prefix.
+            L"Ember HTML Document",  // Browser ProgID description.
+        .direct_launch_url_scheme = "ember",
+        .pdf_prog_id_prefix = L"EmberPDF",  // PDF ProgID prefix.
         .pdf_prog_id_description =
-            L"Chromium PDF Document",  // PDF ProgID description.
+            L"Ember PDF Document",  // PDF ProgID description.
         .active_setup_guid =
-            L"{7D2B3E1D-D096-4594-9D8F-A6667F12E0AC}",  // Active Setup
+            L"{FDFEA5B9-ECF4-4569-9E22-16FDB96870AB}",  // Active Setup
                                                         // GUID.
-        .toast_activator_clsid = {0x635EFA6F,
-                                  0x08D6,
-                                  0x4EC9,
-                                  {0xBD, 0x14, 0x8A, 0x0F, 0xDE, 0x97, 0x51,
-                                   0x59}},  // Toast Activator CLSID.
-        .elevator_clsid = {0xD133B120,
-                           0x6DB4,
-                           0x4D6B,
-                           {0x8B, 0xFE, 0x83, 0xBF, 0x8C, 0xA1, 0xB1,
-                            0xB0}},  // Elevator CLSID.
+        .toast_activator_clsid = {0x44C30F20,
+                                  0x443F,
+                                  0x4682,
+                                  {0xAC, 0x2E, 0xE8, 0x11, 0x65, 0x80, 0x8E,
+                                   0xE5}},  // Toast Activator CLSID.
+        .elevator_clsid = {0xCA4E3C3D,
+                           0xA083,
+                           0x4A17,
+                           {0x90, 0xBF, 0xA5, 0x23, 0xB8, 0x10, 0xA2,
+                            0x6C}},  // Elevator CLSID.
         .elevator_iid = {0xbb19a0e5,
                          0xc6,
                          0x4966,
                          {0x94, 0xb2, 0x5a, 0xfe, 0xc6, 0xfe, 0xd9,
                           0x3a}},  // IElevator IID and TypeLib
         // {BB19A0E5-00C6-4966-94B2-5AFEC6FED93A}.
         .tracing_service_clsid = {0x83f69367,
                                   0x442d,
                                   0x447f,
diff --git a/chrome/installer/mini_installer/mini_installer_constants.cc b/chrome/installer/mini_installer/mini_installer_constants.cc
index c2a03ccb..f4a3a79e 100644
--- a/chrome/installer/mini_installer/mini_installer_constants.cc
+++ b/chrome/installer/mini_installer/mini_installer_constants.cc
@@ -12,7 +12,7 @@ namespace mini_installer {
 // The target name of the installer extracted from resources.
 const wchar_t kSetupExe[] = L"setup.exe";
 // The prefix of the chrome archive resource.
-const wchar_t kChromeArchivePrefix[] = L"chrome";
+const wchar_t kChromeArchivePrefix[] = L"ember";
 // The prefix of the installer resource.
 const wchar_t kSetupPrefix[] = L"setup";
 
@@ -67,12 +67,12 @@ const wchar_t kCleanupRegistryKey[] = L"Software\\Chrome for Testing";
 #else
 // The path to the key containing each app's Clients registry key.
 // No trailing slash on this one because the app's GUID is not appended.
-const wchar_t kClientsKeyBase[] = L"Software\\Chromium";
+const wchar_t kClientsKeyBase[] = L"Software\\Ember";
 // The path to the key containing each app's Client State registry key.
 // No trailing slash on this one because the app's GUID is not appended.
-const wchar_t kClientStateKeyBase[] = L"Software\\Chromium";
+const wchar_t kClientStateKeyBase[] = L"Software\\Ember";
 // The path to the key in which kCleanupRegistryValue is found.
-const wchar_t kCleanupRegistryKey[] = L"Software\\Chromium";
+const wchar_t kCleanupRegistryKey[] = L"Software\\Ember";
 #endif
 
 }  // namespace mini_installer
diff --git a/chrome/installer/setup/installer_crash_reporter_client.cc b/chrome/installer/setup/installer_crash_reporter_client.cc
index 64c13b78..2ff80864 100644
--- a/chrome/installer/setup/installer_crash_reporter_client.cc
+++ b/chrome/installer/setup/installer_crash_reporter_client.cc
@@ -103,7 +103,7 @@ bool InstallerCrashReporterClient::ReportingIsEnforcedByPolicy(bool* enabled) {
       L"SOFTWARE\\Policies\\Google\\Chrome";
 #else
   static const wchar_t kRegistryChromePolicyKey[] =
-      L"SOFTWARE\\Policies\\Chromium";
+      L"SOFTWARE\\Policies\\Ember";
 #endif
   static const wchar_t kMetricsReportingEnabled[] = L"MetricsReportingEnabled";
 
diff --git a/chrome/installer/setup/setup_constants.cc b/chrome/installer/setup/setup_constants.cc
index ddfac630..263c794e 100644
--- a/chrome/installer/setup/setup_constants.cc
+++ b/chrome/installer/setup/setup_constants.cc
@@ -7,14 +7,14 @@
 namespace installer {
 
 // Elements that make up install paths.
-const wchar_t kChromeArchive[] = L"chrome.7z";
-const wchar_t kChromeCompressedArchive[] = L"chrome.packed.7z";
+const wchar_t kChromeArchive[] = L"ember.7z";
+const wchar_t kChromeCompressedArchive[] = L"ember.packed.7z";
 const char kVisualElements[] = "VisualElements";
 const wchar_t kVisualElementsManifest[] = L"chrome.VisualElementsManifest.xml";
 
 // Sub directory of install source package under install temporary directory.
 const wchar_t kInstallSourceDir[] = L"source";
-const wchar_t kInstallSourceChromeDir[] = L"Chrome-bin";
+const wchar_t kInstallSourceChromeDir[] = L"Ember-bin";
 
 const wchar_t kMediaPlayerRegPath[] =
     L"Software\\Microsoft\\MediaPlayer\\ShimInclusionList";
diff --git a/chrome/installer/util/logging_installer.cc b/chrome/installer/util/logging_installer.cc
index 4734595d..8d4a35a6 100644
--- a/chrome/installer/util/logging_installer.cc
+++ b/chrome/installer/util/logging_installer.cc
@@ -126,7 +126,7 @@ base::FilePath GetLogFilePath(const installer::InitialPreferences& prefs) {
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
       FILE_PATH_LITERAL("chrome_installer.log");
 #else  // BUILDFLAG(CHROMIUM_BRANDING)
-      FILE_PATH_LITERAL("chromium_installer.log");
+      FILE_PATH_LITERAL("ember_installer.log");
 #endif
 
   // Fallback to current directory if getting the secure or temp directory
diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
index 0b083561..963913ee 100755
--- a/chrome/tools/build/win/create_installer_archive.py
+++ b/chrome/tools/build/win/create_installer_archive.py
@@ -25,7 +25,7 @@ ARCHIVE_DIR = "installer_archive"
 # suffix to uncompressed full archive file, appended to options.output_name
 ARCHIVE_SUFFIX = ".7z"
 BSDIFF_EXEC = "bsdiff.exe"
-CHROME_DIR = "Chrome-bin"
+CHROME_DIR = "Ember-bin"
 CHROME_PATCH_FILE_SUFFIX = "_patch"  # prefixed by options.output_name
 
 # compressed full archive suffix, will be prefixed by options.output_name
@@ -730,7 +730,7 @@ def _ParseOptions():
         '{BSDIFF|ZUCCHINI}.')
     parser.add_option('-n',
                       '--output_name',
-                      default='chrome',
+                      default='ember',
                       help='Name used to prefix names of generated archives.')
     parser.add_option('--enable_hidpi',
                       default='0',
diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
index 3817a1e5..1919a7b0 100755
--- a/components/policy/tools/generate_policy_source.py
+++ b/components/policy/tools/generate_policy_source.py
@@ -28,7 +28,7 @@ from xml.sax.saxutils import escape as xml_escape
 
 CHROME_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Google\\\\Chrome'
 CHROME_FOR_TESTING_POLICY_KEY = CHROME_POLICY_KEY + ' for Testing'
-CHROMIUM_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Chromium'
+CHROMIUM_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Ember'
 PLATFORM_STRINGS = {
     'chrome_frame': ['win'],
     'chrome_os': ['chrome_os'],
diff --git a/components/policy/tools/template_writers/writer_configuration.py b/components/policy/tools/template_writers/writer_configuration.py
index 283ea725..1bb4794e 100755
--- a/components/policy/tools/template_writers/writer_configuration.py
+++ b/components/policy/tools/template_writers/writer_configuration.py
@@ -27,23 +27,23 @@ def GetConfigurationForBuild(defines):
   if '_chromium' in defines:
     config = {
         'build': 'chromium',
-        'app_name': 'Chromium',
+        'app_name': 'Ember',
         'doc_url': 'https://chromeenterprise.google/policies/',
         'frame_name': 'Chromium Frame',
         'os_name': 'ChromiumOS',
         'webview_name': 'Chromium WebView',
         'win_config': {
             'win': {
-                'reg_mandatory_key_name': 'Software\\Policies\\Chromium',
+                'reg_mandatory_key_name': 'Software\\Policies\\Ember',
                 'reg_recommended_key_name':
-                'Software\\Policies\\Chromium\\Recommended',
-                'mandatory_category_path': ['chromium'],
-                'recommended_category_path': ['chromium_recommended'],
+                'Software\\Policies\\Ember\\Recommended',
+                'mandatory_category_path': ['ember'],
+                'recommended_category_path': ['ember_recommended'],
                 'category_path_strings': {
-                    'chromium': 'Chromium',
-                    'chromium_recommended': 'Chromium - {doc_recommended}',
+                    'ember': 'Ember',
+                    'ember_recommended': 'Ember - {doc_recommended}',
                 },
-                'namespace': 'Chromium.Policies.Chromium',
+                'namespace': 'Ember.Policies.Ember',
             },
             'chrome_os': {
                 'reg_mandatory_key_name': 'Software\\Policies\\ChromiumOS',
@@ -58,9 +58,9 @@ def GetConfigurationForBuild(defines):
                 'namespace': 'Chromium.Policies.ChromiumOS'
             },
         },
-        'admx_prefix': 'chromium',
-        'linux_policy_path': '/etc/chromium/policies/',
-        'bundle_id': 'org.chromium',
+        'admx_prefix': 'ember',
+        'linux_policy_path': '/etc/ember/policies/',
+        'bundle_id': 'dev.v11exe.ember',
     }
   elif '_google_chrome' in defines or '_is_chrome_for_testing_branded' in defines:
     if '_google_chrome' in defines:
