diff --git a/chrome/install_static/chromium_install_modes.h b/chrome/install_static/chromium_install_modes.h
--- a/chrome/install_static/chromium_install_modes.h
+++ b/chrome/install_static/chromium_install_modes.h
@@ -72,16 +72,16 @@ inline constexpr auto kInstallModes = std::to_array<InstallConstants>({
                          {0x94, 0xb2, 0x5a, 0xfe, 0xc6, 0xfe, 0xd9,
                           0x3a}},  // IElevator IID and TypeLib
         // {BB19A0E5-00C6-4966-94B2-5AFEC6FED93A}.
-        .tracing_service_clsid = {0x83f69367,
-                                  0x442d,
-                                  0x447f,
-                                  {0x8b, 0xcc, 0x0e, 0x3f, 0x97, 0xbe, 0x9c,
-                                   0xf2}},  // SystemTraceSession CLSID.
+        .tracing_service_clsid = {0x9C6AB61A,
+                                  0xF2E7,
+                                  0x415E,
+                                  {0x90, 0x8E, 0xC0, 0x7C, 0x93, 0x7D, 0x0E,
+                                   0x88}},  // SystemTraceSession CLSID.
         .tracing_service_iid = {0xa3fd580a,
                                 0xffd4,
                                 0x4075,
                                 {0x91, 0x74, 0x75, 0xd0, 0xb1, 0x99, 0xd3,
                                  0xcb}},  // ISystemTraceSessionChromium IID and
                                           // TypeLib
         .default_channel_name =
             L"",  // Empty default channel name since no update integration.
@@ -97,9 +97,9 @@ inline constexpr auto kInstallModes = std::to_array<InstallConstants>({
         .pdf_doc_icon_resource_index =
             icon_resources::kPDFDocIndex,  // PDF doc icon resource index.
         .sandbox_sid_prefix =
             L"S-1-15-2-3251537155-1984446955-2931258699-841473695-"
             L"1938553385-"
-            L"924012148-",  // App container sid prefix for sandbox.
+            L"1530412577-",  // Stable Ember AppContainer SID family component.
     },
 });
 
